src/main/java/dk/kb/image/AnalyseDeltaEOfPalette.java

```diff
@@ -0,0 +1,47 @@
+package dk.kb.image;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AnalyseDeltaEOfPalette {
```

Comment:
It makes a lot of sense to keep this code in case the palette is changed. However, code that is not used actively in the service should not be under the `src` folder as they make it harder to see what is going on. Could you move it under the `test` folder instead?